### DISCUSSION TITLE
MacOS - Export to PDF generating empty file

### DISCUSSION THREAD

**Discussion by gregfenton** [freeplane/freeplane#364] [Help] [answered]
Freeplane installed via `brew` on a 2019 MacBook Pro (Intel chip).

A normal MM with just nodes and text.
Doing **File >> Export map... >> PDF** and the resulting file is zero bytes in size.
- Have tried multiple times and ways.
- File system has lots of free space

Any guidance as to how to go about triaging?  Is there a way to get to a "console" or a "debug log" when running on MacOS?

**Comment by dpolivaev** [comment, marked as answer]
Try to activate option Preferences…->Environment->Export->Convert texts to shapes in exported PDF. PDFs generated without it are usually much smaller and better but in some cases, Freeplane fails to find font definitions (I still do not understand the exact reason)

**Reply by gregfenton** [reply]
That worked!

Anything I could do to help triage the issue under the other setting?  Is there a log file or something I could be capturing?  Would it be useful to share my `.mm` ?

**Comment by toni2504** [comment]
Hi! I got the same problem as @gregfenton but the solution from @dpolivaev doesn't work for me. Is there anything else I could try?
